src/gui/guiFormSpecMenu.cpp

```diff
@@ -779,6 +780,58 @@ void GUIFormSpecMenu::parseImage(parserData* data, const std::string &element)
 	errorstream<< "Invalid image element(" << parts.size() << "): '" << element << "'"  << std::endl;
 }
 
+void GUIFormSpecMenu::parseAnimatedImage(parserData *data, const std::string &element)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	if (parts.size() != 3 &&
+			(parts.size() < 3 || m_formspec_version <= FORMSPEC_API_VERSION)) {
```

Comment:
Please don't De Morgan this - I would prefer the more obvious condition and the one that is throughout this fileused 

```suggestion
			!(parts.size() > 3 && m_formspec_version > FORMSPEC_API_VERSION)) {
```

Reply:
The current logic also shows that `parts.size() != 3 &&` is redundant.

Reply:
Fine by me, that's more or less what I originally had when I submitted the PR.

Reply:
Were you asked to do this by someone?

Reply:
Sort of? It's in one of the reviews by @DS-Minetest. The main aspect of the review was to reduce nesting though, so this particular mistake is still on me.